code/src/java/pcgen/cdom/facet/analysis/ChallengeRatingFacet.java

```diff
@@ -276,10 +276,7 @@ private Integer getClassRaceCRMod(CharID id, PCClass cl)
 			if (SettingsHandler.getGame().getClassTypeByName(classType) != null)
 			{
 				Integer crMod = raceFacet.get(id).get(MapKey.CRMOD, classType);
-				if (crMod != null)
-				{
-					return crMod;
-				}
+                return crMod;
```

Comment:
Not sure this is a safe change. The if statements usually got added when a null would cause an issue...

Reply:
In the original if crMod is NULL, the IF statement falls through, lines 285-301 have no effect because they are on the other side of an earlier IF, and then NULL is returned anyway by line 302. So this shouldn't change anything.